test/extensions/filters/common/rbac/engine_impl_test.cc

```diff
@@ -22,8 +22,9 @@ namespace {
 
 void checkEngine(const RBAC::RoleBasedAccessControlEngineImpl& engine, bool expected,
                  const Envoy::Network::Connection& connection = Envoy::Network::MockConnection(),
-                 const Envoy::Http::HeaderMap& headers = Envoy::Http::HeaderMapImpl()) {
-  EXPECT_EQ(expected, engine.allowed(connection, headers));
+                 const Envoy::Http::HeaderMap& headers = Envoy::Http::HeaderMapImpl(),
+                 bool isDarkLaunch = false) {
+  EXPECT_EQ(expected, engine.allowed(connection, headers, isDarkLaunch));
```

Comment:
Can we add a test for triggering the dark path?

Reply:
added in rbac_filter_test.cc